### PR TITLE
Correct cache sizes

### PR DESCRIPTION
**Pull request by jonaschn**
@MasterCarl @ArneMayer Could you please double check the cache sizes for the Intel machine? At least the L3 cache size looks suspicious.

### CODE REVIEW

**Comment by ArneMayer** [comment]
image

**Comment by ArneMayer** [comment]
https://ark.intel.com/de/products/75258/Intel-Xeon-Processor-E7-8890-v2-37_5M-Cache-2_80-GHz

**Comment by ArneMayer** [comment]
What do you mean suspicious?

**Comment by jonaschn** [comment]
Ok, but then the cache sizes should be the following according to:
http://www.cpu-world.com/CPUs/Xeon/Intel-Xeon%20E7-8890%20v2.html
```
'L1': 480, # 15x 32 KiB/core
'L2': 3840, # 15x 256 KiB/core
'L3': 38400 # 15x 2,5 MiB/core = 37,5 MiB (shared)
```
But why then are you running just 8 and 16 threads instead of 15 and 30?

File: generate_figures.py [hunk 1]
```diff
@@ -99,8 +100,11 @@ def comma_seperators(x, pos):
 
 if __name__ == '__main__':
     path = sys.argv[1] if len(sys.argv) > 1 else 'benchmark.csv'
-    system_type = sys.argv[2] if len(sys.argv) > 2 else 'power'
-    flags = sys.argv[3:]
+    flags = sys.argv[2:]
```
Comment (MasterCarl) [inline generate_figures.py:103]: This will be obsolete once #46 is merged

**Review by MasterCarl — CHANGES_REQUESTED** [review]

**Comment by jonaschn** [comment]
Close. #48 should be merged instead.

**Pull request closed without merging**